client/store/products/resolvers.js

```diff
@@ -0,0 +1,20 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+const { apiFetch } = wp;
+import { dispatch } from '@wordpress/data';
+import { stringify } from 'qs';
+
+export default {
+	async getProducts( state, query ) {
+		try {
+			const params = query ? '?' + stringify( query ) : '';
```

Comment:
You can use [`stringifyQuery`](https://github.com/woocommerce/wc-admin/blob/fdbfcc994e476f14f794d710a447f16b90023e29/client/lib/nav-utils/index.js#L64) here.